modules/Router.php

```diff
@@ -230,11 +236,28 @@ public static function getPageUrl($page, $content)
     }
 
     /**
+     * Method : getSettings
+     * Description : Returns array of template and data for settings page
```

Comment:
Please make it more descriptive, what "template" is referred here, please explain.